lib/src/bindings/bindings.dart

```diff
@@ -292,6 +299,9 @@ class _ObjectBoxBindings {
     obx_model_last_entity_id =
         _fn<obx_model_last_entity_id_native_t>('obx_model_last_entity_id')
             .asFunction();
+    obx_model_last_index_id =
+        _fn<obx_model_last_index_id_native_t>('obx_model_last_entity_id')
```

Comment:
> 'obx_model_last_entity_id'

That's wrong? This probably explains the error thrown.

Reply:
Thanks, I tend to do that too often, tired eyes.